src/org/zaproxy/zap/extension/search/ExtensionSearch.java

```diff
@@ -298,6 +298,7 @@ public void sessionScopeChanged(Session session) {
 		public void sessionChanged(final Session session) {
 			if (EventQueue.isDispatchThread()) {
 				getSearchPanel().resetSearchResults();
+				getSearchPanel().getSearchStatusLabel().setText("");

```

Comment:
Food for thought:

- Should the search "term" also be reset between loads? (I can see it being handy both ways)
- Do you think there's any point ensuring the search button is in enabled state? (I couldn't get a long enough running search to determine if loading a new session while searching might result in it being disabled.)

Reply:
I'll add the fix keyword after answer/tweaks :zap: